OneTimePasswordLegacy/OTPToken.swift

```diff
@@ -119,18 +119,21 @@ public extension OTPToken {
             return false
         }
         if let keychainItem = self.keychainItem {
-            guard let newKeychainItem = Keychain.sharedInstance.updatePersistentToken(keychainItem,
-                withToken: token) else {
-                    return false
+            do {
+                let newKeychainItem = try Keychain.sharedInstance.updatePersistentToken(keychainItem, withToken: token)
```

Comment:
Length Violation: Line should be 100 characters or less: currently 119 characters